_posts/2022-01-19-7.0.0-release.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: "Apache Arrow 7.0.0 Release"
+date: "2022-01-19 00:00:00 -0600"
+author: pmc
+categories: [release]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+
+The Apache Arrow team is pleased to announce the 7.0.0 release. This covers
+over 3 months of development work and includes [**474 resolved issues**][1]
+from [**?? distinct contributors**][2]. See the Install Page to learn how to
+get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 6.0.1 release, Rémi Dattai and Alessandro Molina have been invited to be committers. 
+Daniël Heres and Yibo Cai have joined the Project Management Committee (PMC). 
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+TBD
+
+## Arrow Flight RPC notes
+
+TBD
+
+## C++ notes
+
+TBD
+
+### Compute Layer
+
+TBD
+
+### Dataset Layer
+
+TBD
+
+### IO and Filesystem Layer
+
+TBD
+
+### Parquet
+
+TBD
+
+## C# notes
+
+TBD
+
+## Go notes
+
```

Comment:
```suggestion
## Go notes

### Arrow

#### Bug Fixes

* License lifted up a level so that it is properly detected for the github.com/apache/arrow/go/v7 module for pkg.go.dev [ARROW-14728](https://github.com/apache/arrow/pull/11715). Documentation on pkg.go.dev will look correct with complete major version handling as of the v7.0.0 release.
* Errors from `MessageReader.Message` get properly surfaced by `Reader.Read` [ARROW-14769](https://github.com/apache/arrow/pull/11739)
* `ipc.Reader` properly uses the allocator it is initialized with instead of making native byte slices [ARROW-14717](https://github.com/apache/arrow/pull/11712)
* Fixed a CI issue where the CGO tests were crashing on windows [ARROW-14589](https://github.com/apache/arrow/pull/11611)
* Various fixes for internal usages of `Release` and `Retain` to maintain proper management of reference counting.

#### Enhancements

* Continuous Integration for Go library now uses Go1.16 as the version being tested [ARROW-14985](https://github.com/apache/arrow/pull/11860)
* `ValueOffsets` function added to `array.String` to return the entire slice of offsets [ARROW-14645](https://github.com/apache/arrow/pull/11653)
* `array.Interface` has been lifted to `arrow.Array`, `array.{Record,Column,Chunked,Table}` have been lifted to `arrow.{Record,Column,Chunked,Table}`. Interface `arrow.ArrayData` has been created to be used instead of `array.Data`. Aliases have been provided for the `array` package so existing code that doesn't directly use `array.Data` shouldn't be affected. The aliases will be removed in v8. [ARROW-5599](https://github.com/apache/arrow/pull/11832)
* Arrays and Records now support marshalling to JSON via the `json.Marshaller` interface. Builders support adding values to them by unmarshalling from JSON via the `json.Unmarshaller` interface. `array.FromJSON` function added to create Arrays from JSON directly. [ARROW-9630](https://github.com/apache/arrow/pull/11359)
* Basic handling of field referencing and expression building similar to the C++ Compute APIs added through the new `compute` package in preparation for adding compute interfaces. Does not yet allow *executing* expressions. [ARROW-14430](https://github.com/apache/arrow/pull/11514)

### Parquet

#### Enhancements

* Updated dependency versions [ARROW-14462](https://github.com/apache/arrow/pull/11537)
* `file` module added, Go Parquet library now supports full file reading and writing. [ARROW-13984](https://github.com/apache/arrow/pull/11146) [ARROW-13986](https://github.com/apache/arrow/pull/11538). Does not yet provide direct Parquet <--> Arrow conversions.
```